packages/api/src/external/commonwell-v2/patient/patient.ts

```diff
@@ -116,23 +118,10 @@ export async function registerAndLinkPatientInCwV2({
     await sleep(waitTime);
```

Comment:
I'm thinking we should move this inside `runPatientLinkingWithRetries`, because we prob want to wait in between link iterations as well, right?